src/main/resources/db/dataset/init-demo-data.sql

```diff
@@ -498,13 +515,16 @@ INSERT INTO public.person (id, email, first_name, last_name, birthdate, title, p
     (63, 'test@test.com', 'Opre',        'Vious',       315532800000, 0, TRUE,  '11', true, 'PREVIOUS'),
     (64, 'test@test.com', 'Agrippa',     'Nel',         1024815788000, 0, FALSE,  '11', true, 'PREVIOUS'),
     (65, 'test@test.com', 'Isidore',     'Champ',       1070870588000, 0, FALSE,  '11', false, 'PREVIOUS'),
-    (66, 'test@test.com', 'Bowie',       'Ken',         null,          0, FALSE,  '20', false, 'NEXT');
-
+    (66, 'test@test.com', 'Robert',      'Diggs',       null,          0, FALSE,  '24', false, 'PREVIOUS'),
+    (67, 'test@test.com', 'Clifford',    'Smith',       1024815788001, 0, FALSE,  '24', false, 'PREVIOUS'),
+    (68, 'test@test.com', 'Gary',        'Grice',       1024815788002, 0, FALSE,  '24', false, 'NEXT'),
+    (69, 'test@test.com', 'Dennis',      'Coles',       null,          0, FALSE,  '24', false, 'NEXT');
```

Comment:
maybe having a panel person and a privileged person could be usefull to cover all cases